ubuntu/get-docker.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/sh
+
+
+if ( apt-cache show ca-certificates )
+then
+  echo "-> ca-certificates is in packages list"
+else
+  echo "-> cannot find ca-certificates, updating package list"
+  sudo DEBIAN_FRONTEND=noninteractive apt update
+fi 
+
+if ( apt-cache show curl )
+then
+  echo "-> curl is in packages list"
+else
+  echo "-> cannot find curl, updating packages list"
+  sudo DEBIAN_FRONTEND=noninteractive apt update
+fi
+
+if ( apt-cache show gnupg )
+then
+  echo "-> gnupg is in packages list"
+else
+  echo "-> cannot find gnupg, updating packages list"
+  sudo DEBIAN_FRONTEND=noninteractive apt update
+fi
+
+if [ -f /etc/ssl/certs/ca-certificates.crt ]
+then
+  echo "-> /etc/ssl/certs/ca-certificates.crt exists"
+else
+  echo "-> installing ca-certificates"
+  sudo apt install ca-certificates
```

Comment:
Depending on what version of Ubuntu, this is now a "fake package" that was just there for backwards compatibility and is now just included in the `apt` package:

https://packages.ubuntu.com/lunar/apt-transport-https

So you probably don't need it, but if it's not breaking anything you can leave it in.

Reply:
i saw that it was there already, but didnt look deeper.